app/src/main/java/org/sil/gatherwords/room/Meaning.java

```diff
@@ -0,0 +1,31 @@
+package org.sil.gatherwords.room;
+
+import android.arch.persistence.room.Entity;
+import android.arch.persistence.room.ForeignKey;
+import android.arch.persistence.room.Index;
+import android.arch.persistence.room.PrimaryKey;
+
+/**
+ * Entity class for each word of a session
+ */
+@Entity(foreignKeys = @ForeignKey(entity = Word.class,
+                                  parentColumns = "id",
+                                  childColumns = "wordID",
+                                  onDelete = ForeignKey.CASCADE,
+                                  onUpdate = ForeignKey.CASCADE),
+        indices = @Index("wordID"))
+public class Meaning {
```

Comment:
What are these meanings? Are they senses (e.g., bank = financialInstitution, riverSide, turn) or meanings written in other languages?

Reply:
Dropped a TODO.